fields_definitions.go

```diff
@@ -244,11 +244,13 @@ func (f *TimeStampField) WriteSQL(sqlBuf *SqlBuffer) {
 
 	if f.NotNull {
 		sqlBuf.WriteString(" NOT NULL")
+	} else {
+		sqlBuf.WriteString(" NULL")
 	}
 
 	if f.Default != nil {
 		sqlBuf.WriteString(" DEFAULT ")
-		sqlBuf.WriteValue(*f.Default)
+		sqlBuf.WriteString(*f.Default)
```

Comment:
It's generated file, don't change it manually. Fix https://github.com/go-qbit/storage-mysql/blob/master/gen/fields.go and generate it.
Value must be quoted because it can have dangerous characters.